<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LoadGraphDialog</class>
 <widget class="QDialog" name="LoadGraphDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>406</width>
    <height>143</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="graphURILabel">
     <property name="text">
      <string>Graph URI:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="QLineEdit" name="graphURIEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="layerNameLabel">
     <property name="text">
      <string>Layer Name:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <widget class="QLineEdit" name="layerNameEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="loadFromURIButton">
     <property name="text">
      <string>Load From URI</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="OrLabel">
     <property name="text">
      <string>OR</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QPushButton" name="loadFromFileButton">
     <property name="text">
      <string>Load From File</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
